target/linux/rtl838x/files-5.4/drivers/mtd/mtdsplit/mtdsplit.c

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2009-2013 Felix Fietkau <nbd@nbd.name>
+ * Copyright (C) 2009-2013 Gabor Juhos <juhosg@openwrt.org>
+ * Copyright (C) 2012 Jonas Gorski <jogo@openwrt.org>
+ * Copyright (C) 2013 Hauke Mehrtens <hauke@hauke-m.de>
```

Comment:
Did you explain somewhere why you don't just use the generic mtdsplit.c split driver?

Reply:
File was left-over from testing. Removed.